breaking-change-detection/README.md

```diff
@@ -0,0 +1,44 @@
+# Breaking change detection
+
+> You need to have the [`buf` CLI][install] installed to follow along with this example.
+
+This project shows Buf [breaking change detection][breaking] in action. There are three different [Buf inputs][inputs] in play here:
+
+* An [`initial`](./initial/acme/weather/v1/weather.proto) input that serves as the basis for comparison.
+* An input called [`compatible`](./compatible/acme/weather/v1/weather.proto) that introduces **non-breaking** changes to `initial`. Inline comments indicate changes.
+* An input called [`incompatible`](./incompatible/acme/weather/v1/weather.proto) that introduces **breaking** changes to `initial`. Inline comments indicate changes.
+
+All three inputs are for the same `acme.weather.v1` package. Each input uses the default [`FILE`][file] breaking change rule, which detects breaking changes on a per-file basis and also verifies compatibility in how Protobuf definitions act across the wire (`WIRE`) and in the JSON representation of the definitions (`WIRE_JSON`).
```

Comment:
Considering both the input and against have a breaking config, which one would be used? I believe its the input one, but given that its in all and the same everywhere I feel like this maybe worth calling out as it isn't clear from the example.

Reply:
Okay, I've added a footnote here